src/framework/core/js/DataBinding.js

```diff
@@ -1507,4 +1507,76 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         return that;
     };
 
+    /**
+     * Calculates the changes between the model values 'value' and
+     * 'oldValue'. The optional argument 'changePathPrefix' is prepended
+     * to the generated change paths. This is useful for generating
+     * change records to be applied at a non-root path in a model.
+     * Returns an array of change records, suitable for passing to
+     * fluid.fireChanges().
+     */
```

Comment:
Some Javadoc-style parameter comments, as we sometimes see elsewhere,  would be useful here - e.g. @param changePathPrefix {String}